src/Adapter/Doctrine/DoctrineCachePool.php

```diff
@@ -132,6 +132,13 @@ protected function removeListItem($name, $key)
                 unset($list[$i]);
             }
         }
+
+        if ($list === []) {
+            $this->removeList($name);
```

Comment:
Why can't we save an empty list? Do we have to remove it?

Reply:
This was a while ago. I don't really recall I believe is simply a matter of cleaning up behind.
With backend like memcached, this can can help avoid some pruning cycle that could potentially delete some cache entries that are desired.